client/src/utils/dates/Date.js

```diff
@@ -1,8 +1,19 @@
-export function isDate() {
-  const today = new Date();
-  const date = `${today.getDate()}.${
-    today.getMonth() + 1
-  }.${today.getFullYear()}`;
-
-  return date;
-}
+// Unix Timestamps
+export const isUnixTime = () => Math.floor(Date.now() / 1000);
+
+// date-format: DD.MM.YYYY
+export const isDate = (today = new Date()) =>
+  `${("0" + today.getDate()).slice(-2)}.${("0" + (today.getMonth() + 1)).slice(
+    -2
+  )}.${today.getFullYear()}`;
```

Comment:
```suggestion
export const isDate = (today = new Date().toLocaleDateString("de-DE")
```

This should do exactly the same as you did there.

Reply:
jap right, i used that in the server side, discovered it later, after finishing the server side, i will refactore and rebuild the client again, sorry for the mess